drivers/charger/CMakeLists.txt

```diff
@@ -8,3 +8,4 @@ zephyr_library_sources_ifdef(CONFIG_CHARGER_MAX20335 charger_max20335.c)
 zephyr_library_sources_ifdef(CONFIG_SBS_CHARGER sbs_charger.c)
 zephyr_library_sources_ifdef(CONFIG_USERSPACE charger_handlers.c)
 zephyr_library_sources_ifdef(CONFIG_EMUL_SBS_CHARGER emul_sbs_charger.c)
+zephyr_library_sources_ifdef(CONFIG_CHARGER_BQ2407X bq2407x.c)
```

Comment:
alphabetical order please (I know that CONFIG_ENUM is wrong but let's try to keep it sorted since now...).

Reply:
done